src/module.ts

```diff
@@ -63,7 +65,7 @@ export default defineNuxtModule<ModuleOptions>({
       const as = prefix
         ? prefix + (options.upperAfterPrefix ? radash.pascal(alias) : alias)
         : alias;
-      addImports({ name, as, from: "radash" });
+      addImports({ name, as, from: resolve("./runtime/radash") });
```

Comment:
Tried to use `findPath` + `resolve` from `nuxt/kit` to directly resolve `radash` here, but it seems that it does not work.